WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -59,6 +59,17 @@ extension WordPressAuthenticationManager {
         // Ref https://github.com/wordpress-mobile/WordPress-iOS/pull/12332#issuecomment-521994963
         let enableSignInWithApple = !(BuildConfiguration.current ~= [.a8cBranchTest, .a8cPrereleaseTesting])
 
+        let googleLogingWithoutSDK: Bool = {
+            switch BuildConfiguration.current {
+            case .appStore:
+                // Rely on the remote flag in production
+                return RemoteFeatureFlagStore().value(for: FeatureFlag.sdkLessGoogleSignIn)
+            case _:
+                // Do whatever is hardcoded or overridden on device in non-production builds
+                return FeatureFlag.sdkLessGoogleSignIn.enabled
+            }
+        }()
```

Comment:
@hassaanelgarem @momo-ozawa you've been working on feature flags right? 

Is this a reasonable approach? I was surprised by having to use a dedicated store to read remote values instead of accessing them directly from `FeatureFlag.<case>.enabled` so I'm not sure whether I'm using it right.

I considered moving this logic into the `enabled` implementation, but realized it would have created an infinite loop in case the value was not found remotely 😅

Reply:
>  I was surprised by having to use a dedicated store to read remote values instead of accessing them directly from FeatureFlag.<case>.enabled

Yes, using remote feature flags is really unintuitive at the moment, but coincidently, this is something I'm working on fixing during HACK week (we're having a late HACK week within our team cause we were on release rotation during the original week). Ref: pbArwn-5Uc-p2#comment-7452

> Is this a reasonable approach?

Yes, this looks reasonable. However, you could also ditch the whole switch and only depend on `RemoteFeatureFlagStore.value(for:)`. Currently, `value(for:)` returns the overridden value if it exists. If not, it returns the remote value if it exists. If not, it returns the default (hardcoded) value. If you've added the switch just to facilitate testing, you can consider removing it. If it serves a different purpose, then this looks good 👍 

FYI: There's a known issue where if you have a remote flag that is remotely set to `true`, you can't override it to return `false`. This is also something I'm working on fixing now.

Reply:
Thank you for the feedback @hassaanelgarem 🙇‍♂️ 

> However, you could also ditch the whole switch and only depend on `RemoteFeatureFlagStore.value(for:)`.

Sweet. I'll do that.

Reply:
> I'll look into it and update if necessary.

For the record, I've rebased and adopted the new syntax vai [`e91ca3c`](https://github.com/wordpress-mobile/WordPress-iOS/pull/20128/commits/e91ca3cd61339592cbba4d5de2db6bc05f63bb6c)